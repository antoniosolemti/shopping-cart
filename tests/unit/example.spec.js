import { shallowMount } from '@vue/test-utils'
import Header from '@/components/Header.vue'

describe('Header.vue', () => {
  it('renders header', () => {
    const msg = 'new message'
    const wrapper = shallowMount(Header, {
      stubs:['router-link']
    })
    expect(wrapper.text()).toMatchSnapshot(msg)
  })
})
