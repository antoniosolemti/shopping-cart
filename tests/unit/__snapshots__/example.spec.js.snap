// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue renders header: new message 1`] = `
"Home |
    About"
`;
